### ISSUE TITLE
PrintNanny Cam shows incorrect aspect ratio

### ISSUE THREAD

**Issue by leigh-johnson** [bitsy-ai/printnanny-os#244]
## **Describe the bug**
Looks like the aspect ratio displayed in Mission Control > Camera is not correct. I'm guessing this is just a video element issue, since the OctoPrint HLS stream is displayed with the correct aspect ratio.

## **Expected behavior**
PrintNanny Cam display the correct aspect ratio

## **Screenshots**
![unnamed (3)](https://user-images.githubusercontent.com/2601819/216780450-6385bf2c-09e6-4d0c-8371-99f7ec72769c.png)
![unnamed (2)](https://user-images.githubusercontent.com/2601819/216780455-4b9c7ebd-66f1-4162-b9c5-6575224a79e6.png)

## Issue.txt

```
-----------------------
Build Info:  |
-----------------------
IMAGE_NAME = printnanny-release-image-raspberrypi4-64-20230202052633
DISTRO_PRETTY = PrintNanny Linux 0.6.2 (Feldspar)
VARIANT_NAME = PrintNanny OS Core
VARIANT_ID = core
DISTRO = printnanny
DISTRO_NAME = PrintNanny Linux
DISTRO_VERSION = 0.6.2
DISTRO_CODENAME = Feldspar
ID_LIKE = BitsyLinux
HOME_URL = https://printnanny.ai
BUG_REPORT_URL = https://github.com/bitsy-ai/printnanny-os/issues
YOCTO_VERSION = 4.1.2
YOCTO_CODENAME = Langdale
MACHINE = raspberrypi4-64
TUNE_PKGARCH = cortexa72
-----------------------
Layer Revisions:      |
-----------------------
meta              = langdale:f59aa3752dbc5552469207894bea9c01bd68fb6e
meta-poky         = langdale:f59aa3752dbc5552469207894bea9c01bd68fb6e
meta-yocto-bsp    = langdale:f59aa3752dbc5552469207894bea9c01bd68fb6e
meta-raspberrypi  = langdale-libcamera-fix:99f5a91d9d4b34ff8c214cf61cd12f375da1a76a
meta-oe           = langdale:f8cb46d803190bb02085c8a7d20957a71d32f311
meta-python       = langdale:f8cb46d803190bb02085c8a7d20957a71d32f311
meta-multimedia   = langdale:f8cb46d803190bb02085c8a7d20957a71d32f311
meta-bitsy        = main:3483ea5157e7d612154d7cda67715c11056ddabf
meta-neural-network = mickledore:f19fa74c2f4c868eda1c48d709b66848f9cca5e8
meta-printnanny   = main:3483ea5157e7d612154d7cda67715c11056ddabf
meta-networking   = langdale:f8cb46d803190bb02085c8a7d20957a71d32f311
meta-filesystems  = langdale:f8cb46d803190bb02085c8a7d20957a71d32f311
meta-initramfs    = langdale:f8cb46d803190bb02085c8a7d20957a71d32f311
meta-webserver    = langdale:f8cb46d803190bb02085c8a7d20957a71d32f311
meta-swupdate     = mickledore:c47a10be18412919f4c7041be05509ba97ac79f4
meta-microcontroller = mickledore:a848733698649628d0c3e34ae04ced0ee6ce3aee
```

**Comment by leigh-johnson** [comment]
:eyes: This has to be an HTML display issue. `chrome://webrtc-internals/` shows the correct width for the rtp stream.
```
Statistics ITv11V504631133
--
timestamp | 2/7/2023, 2:48:10 PM
ssrc | 504631133
kind | video
trackId | DEPRECATED_TI3
transportId | Tv11
codecId | CITv11_96_level-asymmetry-allowed=1;packetization-mode=0;profile-level-id=42e01f
[codec] | H264 (96, level-asymmetry-allowed=1;packetization-mode=0;profile-level-id=42e01f)
mediaType | video
jitter | 0.001
packetsLost | 0
trackIdentifier | janusv1
mid | v1
packetsReceived | 89864
[packetsReceived/s] | 86.99431694916628
bytesReceived | 87001003
[bytesReceived_in_bits/s] | 699050.3333551075
headerBytesReceived | 2156736
[headerBytesReceived_in_bits/s] | 16702.908854239926
lastPacketReceivedTimestamp | 1675810090216
[lastPacketReceivedTimestamp] | 2/7/2023, 2:48:10 PM
jitterBufferDelay | 460.147
[jitterBufferDelay/jitterBufferEmittedCount_in_ms] | 25.9999999999998
jitterBufferEmittedCount | 16623
framesReceived | 16624
[framesReceived/s] | 15.998954841225983
[framesReceived-framesDecoded-framesDropped] | 0
frameWidth | 640
frameHeight | 480
framesPerSecond | 16
framesDecoded | 16624
[framesDecoded/s] | 15.998954841225983
keyFramesDecoded | 279
[keyFramesDecoded/s] | 0.9999346775766239
framesDropped | 0
totalDecodeTime | 31.138603
[totalDecodeTime/framesDecoded_in_ms] | 1.8481249999999783
totalProcessingDelay | 308.51581799999997
[totalProcessingDelay/framesDecoded_in_ms] | 17.51762499999998
totalAssemblyTime | 27.925553999999998
[totalAssemblyTime/framesAssembledFromMultiplePackets_in_ms] | 1.6456874999999371
framesAssembledFromMultiplePackets | 16624
totalInterFrameDelay | 1038.5850000000014
[totalInterFrameDelay/framesDecoded_in_ms] | 62.5
totalSquaredInterFrameDelay | 64.91119499999834
[interFrameDelayStDev_in_ms] | 0.4999999993688107
pauseCount | 0
totalPausesDuration | 0
freezeCount | 0
totalFreezesDuration | 0
decoderImplementation | FFmpeg
firCount | 0
pliCount | 0
nackCount | 0
qpSum | 332481
[qpSum/framesDecoded] | 20
powerEfficientDecoder | false

</slot>
```

**Comment by leigh-johnson** [comment]
:+1: LGTM! This fix will be available in v0.6.4

![Screenshot from 2023-02-07 16-10-04](https://user-images.githubusercontent.com/2601819/217395697-2818fda3-9f73-4c2c-bef6-23c9720434c0.png)

I'll re-think the way calibration notifications are delivered while building workflows for v0.7.0. The pop-up notifications in Mission Control aren't super helpful, because it's not clear what action should be taken.
